chromium.sh

```diff
@@ -9,6 +9,17 @@ get_int32_property() {
     | awk 'match($0, /uint32 ([0-9]+)/, m){print m[1];}'
 }
 
+merge_extensions() {
+  (
+    shopt -s nullglob
+    dest=/app/chromium/unified-extensions/$1
+    mkdir -p $dest
+    for ext in /app/chromium/$1/*; do
+      ln -s $ext $dest
+    done
```

Comment:
This doesn't work because `/app` is read-only while the app runs:

```
wjt@camille:~/.local/share$ flatpak run org.chromium.Chromium//test
mkdir: cannot create directory ‘/app/chromium/unified-extensions/extensions’: Read-only file system
ln: failed to create symbolic link '/app/chromium/unified-extensions/extensions': Read-only file system
ln: failed to create symbolic link '/app/chromium/unified-extensions/extensions': Read-only file system
mkdir: cannot create directory ‘/app/chromium/unified-extensions/native-messaging-hosts’: Read-only file system
mkdir: cannot create directory ‘/app/chromium/unified-extensions/policies’: Read-only file system
mkdir: cannot create directory ‘/app/chromium/unified-extensions/policies’: Read-only file system
```

Reply:
Weird, the extension mount point `/app/chromium/unified-extensions` should be tmpfs for the merge-dirs thing to work. Maybe a tmpfs is mounted over it only when some actual extensions are installed?

Reply:
Huh...yeah, I was under the assumption this would be a tmpfs always, but looking at the source, it only mounts the tmpfs when it goes to mount an extension...so it never will if no extension under that point is available. That's definitely a bit of a curveball, I guess I could just throw it up under something guaranteed to be a private tmpfs (`/run/user/1000/app/...` or `/tmp`), it's just a bit uglier that way.

Reply:
An idea: we can create symlinks `/app/chromium/unified-extensions/policies` -> `/app/chromium/policies/policies` at build time, so that the symlinks appear in the sandbox when there are no combined extension mounted, and in the wrapper do a check if `mountpoint /app/chromium/unified-extensions` - then, if tmpfs is mounted over the dir, do the symlink merge.